### ISSUE TITLE
Notification Component color contrast of is-warning does not meet WCAG criteria

### ISSUE THREAD

**Issue by Sylphias** [GovTechSG/sgds#34]
Yellow and white text combo on Notification does not meet WCAG 4.5 on color contrast checker.

#F5A623 - Yellow and #FFFFFF - White
![image](https://user-images.githubusercontent.com/5444382/62677689-c5debd80-b9e1-11e9-9ef5-206483a7db8e.png)
